src/machines/commerce-machine.ts

```diff
@@ -163,7 +171,42 @@ export const commerceMachine = createMachine<
         (_, event) => {
           if (event.type !== 'done.invoke.fetchPricingDataService') return {}
 
-          return {pricingData: event.data}
+          const extractAppliedDefaultCoupon = (
```

Comment:
If the pricing data has a default (site-wide) coupon that has been applied, this will add that to the machine context as `couponToApply` so that it gets passed to the Stripe Checkout Session.